airflow/providers/apache/hive/CHANGELOG.rst

```diff
@@ -24,6 +24,24 @@
 Changelog
 ---------
 
+6.1.1
+.....
+
+* ``Add Python 3.11 support (#27264)``
```

Comment:
Might be misleading because Hive is the only operator that does not support Pyhon 3.11. In case of Hive, it is really "add a way how to disable Python 3.11 support per provider.

Reply:
I wonder why we have no "Remove Python 3.7 support as welll" ?

Reply:
Ah - it's because there was no "actual" change in the provider - only in the generated setup.py. I wonder if we should maybe add the "remove python 3.7 support" everywhere ?

Reply:
https://github.com/apache/airflow/pull/32015